### ISSUE TITLE
Add a feature for sending deployment status notifications.

### ISSUE THREAD

**Issue by yezyilomo** [yezyilomo/github-auto-deployer#14]

**Comment by codsane** [comment]
May add this to my fork soon, in the form of either push notifications or Discord webhooks (as I'm using the project to auto-deploy a Discord bot at the moment).

Discord webhooks aren't the best choice for everyone so I also would love to add something like [apprise](https://github.com/caronc/apprise) if it existed in JS. For now I'm thinking [Pushover](https://www.npmjs.com/package/pushover-notifications) and Discord webhooks.

**Comment by yezyilomo** [comment]
That would be very cool, [apprise](https://github.com/caronc/apprise) looks amazing, they have integrated many services.

**Comment by codsane** [comment]
Yeah it is a very neat project. I just noticed they have a simple CLI tool included: https://github.com/caronc/apprise#command-line

I think I'll just use that, no need to find a JS replacement then!

**Comment by yezyilomo** [comment]
Yeah the global config could be done in `app.conf` and per repo in `deployment.yml` then use shelljs to execute commands to send notifications. In some ways JS replacement makes sense because to use `apprise` one would require to install Python on their server, this might be a stopper for some people especially if their projects doesn't require Python because it might seem like it requires a lot of dependencies.
